### PR TITLE
Bump windows-process-tree

### PR DESCRIPTION
**Pull request by rzhao271**

### CODE REVIEW

File: src/typings/windows-process-tree.d.ts
Comment (Tyriar) [inline src/typings/windows-process-tree.d.ts:1]: Why is this needed?

Comment (rzhao271) [reply src/typings/windows-process-tree.d.ts:1]: I was hoping to add in the file for windowsShellHelper.ts:12.
I could be understanding the import system wrong, but would the alternative be to move windows-process-tree from being an optional dependency to being a regular dependency?

Comment (rzhao271) [reply src/typings/windows-process-tree.d.ts:1]: Another reason this types file is needed is because `@types/windows-process-tree` contains types for `windows-process-tree` instead of `@vscode/windows-process-tree`.

Comment (Tyriar) [reply src/typings/windows-process-tree.d.ts:1]: Don't we need to fix that in the module then? The typings wouldn't work for anyone would they?

Comment (rzhao271) [reply src/typings/windows-process-tree.d.ts:1]: My reasoning above doesn't work. Let me try again:
1. The types module is actually deprecated. https://www.npmjs.com/package/@types/windows-process-tree
2. `@vscode/windows-process-tree` is an optional dependency because it's not installed on all OSes.
3. Types information for it is still being requested in `windowsShellHelper.ts`. I believe it's also requested by TypeScript in `noImplicitAny` contexts, though I have to double-check that.

Comment (Tyriar) [reply src/typings/windows-process-tree.d.ts:1]: Oh I see, in that case can we add a comment before `declare module '@vscode/windows-process-tree'` explaining that this needs to be there because it's an optional dependency?

**Review by joaomoreno — APPROVED** [review]
